ccdproc/core.py

```diff
@@ -165,6 +165,10 @@ def ccd_process(ccd, oscan=None, trim=None, error=False, master_bias=None,
         If True, scale the dark frame by the exposure times.
         Default is ``False``.
 
+    gain_corrected : bool, optional
```

Comment:
I'm not exactly sure that's the best approach because the name could also mean that the "object" has been gain corrected.